TidepoolKit Example/RootTableViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  RootTableViewController.swift
+//  TidepoolKit Example
+//
+//  Created by Darin Krauss on 1/10/20.
+//  Copyright © 2020 Tidepool Project. All rights reserved.
+//
+
+import os.log
+import UIKit
+import TidepoolKit
+import TidepoolKitUI
+
+class RootTableViewController: UITableViewController {
+    private let api = TAPI()
+    private var environment: TEnvironment?
+    private var session: TSession? {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    private enum Section: Int, CaseIterable {
+        case status
+        case authentication
+    }
+
+    private enum Authentication: Int, CaseIterable {
+        case login
+        case refresh
+        case logout
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return Section.allCases.count
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch Section(rawValue: section)! {
+        case .status:
+            return NSLocalizedString("Status", comment: "The title for the header of the status section")
+        case .authentication:
+            return NSLocalizedString("Authentication", comment: "The title for the header of the authentication section")
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch Section(rawValue: section)! {
+        case .status:
+            return 1
+        case .authentication:
+            return Authentication.allCases.count
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch Section(rawValue: indexPath.section)! {
+        case .status:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "StatusTableViewCell", for: indexPath) as! StatusTableViewCell
```

Comment:
The `IdentifiableClass` protocol is provided by `LoopKit` and this project does not have access to `LoopKit` (by design since this is not a Loop-specific project). I think this is fine for this example class.